core/sessions/ldapauth/ldap.go

```diff
@@ -0,0 +1,861 @@
+package ldapauth
+
+/*
+
+The LDAP authentication module forwards the credentials in the user session request
```

Comment:
nit/ go not rust :stuck_out_tongue_winking_eye: 
```suggestion
The LDAP authentication package forwards the credentials in the user session request
```

Reply:
Done!